app/config/collections/platform.php

```diff
@@ -330,6 +330,72 @@
                 'default' => null,
                 'array' => false,
                 'filters' => ['datetime'],
+            ],
+            [
+                '$id' => ID::custom('authSubscriptionsEnabled'),
```

Comment:
It might be easier here to follow what we do for auth and just have a single json attribute for `payments` containing all the configs we need.